foyer/forcefields/xml/oplsaa.xml

```diff
@@ -174,25 +174,25 @@
   <Type name="opls_168" class="HO" element="H" mass="1.008" def="H[O;%opls_167]" overrides="opls_155" doi="10.1021/ja9621760"/>
   <Type name="opls_169" class="opls_169" element="O" mass="15.9994"/>
   <Type name="opls_170" class="opls_170" element="H" mass="1.008"/>
-  <Type name="opls_171" class="opls_171" element="O" mass="15.9994"/>
-  <Type name="opls_172" class="opls_172" element="H" mass="1.008"/>
-  <Type name="opls_173" class="opls_173" element="C" mass="12.011"/>
-  <Type name="opls_174" class="opls_174" element="C" mass="12.011"/>
+  <Type name="opls_171" class="OH" element="O" mass="15.9994" def="[O;X2]([%opls_173,%opls_174])(H)" desc="O in triols" overrides="opls_154" doi="10.1021/ct200731v"/>
+  <Type name="opls_172" class="HO" element="H" mass="1.008" def="[H;X1]([O;%opls_171])" desc="H(O) in triols" overrides="opls_155" doi="10.1021/ct200731v"/>
+  <Type name="opls_173" class="CT" element="C" mass="12.011" def="[C;X4]([H])([H])([O;X2]([H]))([C;X4]([O;X2]([H])))" desc="C(H2)(OH) in triols" overrides="opls_157, opls_158" doi="10.1021/ct200731v"/>
+  <Type name="opls_174" class="CT" element="C" mass="12.011" def="[C;X4]([H])([O;X2]([H]))([C;X4]([O;X2]))([C;X4]([O;X2]))" desc="C(HROH): triols" overrides="opls_158" doi="10.1021/ct200731v"/>
```

Comment:
This one will match a triol, so that's good. Is this the central COH in propane-1,2,3-triol?

Reply:
Yes, I added an extra bit of info to the description to make that more clear.